shared/js/background/utils.js

```diff
@@ -440,12 +440,21 @@ export function getInstallTimestamp (atb) {
  * @returns {boolean}
  */
 export function isInstalledWithinDays (numberOfDays, fromDate = Date.now(), atb = settings.getSetting('atb')) {
-    if (!atb) return false
+    return daysInstalled(fromDate, atb) <= numberOfDays
+}
+
+/**
+ * Returns the days since installed using atb
+ * @param {number} [fromDate]
+ * @param {string} [atb]
+ * @returns {number}
+ */
+export function daysInstalled (fromDate = Date.now(), atb = settings.getSetting('atb')) {
+    if (!atb) return NaN
 
     const installTimestamp = getInstallTimestamp(atb)
     // If we can't get the install date, assume it wasn't installed in time period
-    if (!installTimestamp) return false
+    if (!installTimestamp) return NaN
```

Comment:
I'm returning `NaN` here and above so I can keep the type consistency while also ensuring maximum flexibility. `NaN` is both falsy and returns false for all math comparisons, so if you do `5 < NaN` or `5 > NaN` they're both false, which match what you would expect in these cases.

Reply:
Hot new type just dropped.